.pre-commit-config.yaml

```diff
@@ -23,6 +23,8 @@ repos:
     rev: 21.12b0
```

Comment:
Can you please update `black` to the latest version please? It would also fix this problem.

Reply:
Sure, I was just worried it would reformat a whole bunch of stubs. I'll do it later today

Reply:
Done. Only resulted in two very minor whitespace changes.